lib/browser/api/browser-window.js

```diff
@@ -141,7 +141,7 @@ BrowserWindow.fromId = (id) => {
 }
 
 BrowserWindow.getAllWindows = () => {
-  return TopLevelWindow.getAllWindows().filter(isBrowserWindow)
+  return TopLevelWindow.getAllWindows()
```

Comment:
This seems significant.

`semver/minor`?

Reply:
Oh this is a mis-commit, will remove